### PR TITLE
Page wrapper

### PR DESCRIPTION
**Pull request by Dantemss**
Should all work, but expect broken specs for now.

### CODE REVIEW

**Comment by Dantemss** [comment]
Feel free to review, I'll work on the specs tomorrow.

File: app/assistants/i_reading_assistant.rb [hunk 1]
```diff
@@ -57,173 +24,18 @@ def self.schema
...
-
   def self.distribute_tasks(task_plan:, taskees:)
     title = task_plan.title || 'iReading'
     opens_at = task_plan.opens_at
     due_at = task_plan.due_at || (task_plan.opens_at + 1.week)
 
-    task_step_attributes = split_task_plan_content(task_plan)
+    page_ids = task_plan.settings['page_ids']
+    cnx_pages = page_ids.collect do |page_id|
+      page = Content::Api::GetPage.call(id: page_id).outputs.page
```
Comment (jpslav) [inline app/assistants/i_reading_assistant.rb]: I thought we were going to make `GetPage` return an `OpenStax::Cnx::V1::Page`, right?  (this was how @kevinburleigh75 was going to stub this routine to return the `Page` objects he built manually.

Comment (Dantemss) [inline app/assistants/i_reading_assistant.rb]: Sure I'll change it.

**Comment by Dantemss** [comment]
Merging for Kevin

**Pull request merged**